## Calibration frame pose
# This yaml specified the position of the checkerboard frame (CB)
# compared to the panda_link0 frame.

## Calibration frame position ##
x_pos: 1        # Relative position on the x axis [m]
y_pos: 0        # Relative position on the y axis [m]
z_pos: 0        # Relative position on the z axis [m]
yaw: deg(270)    # Rotation around the z axis [deg]
pitch: deg(0)  # Rotation around the y axis [deg]
roll: deg(0)   # Rotation around the x axis [deg]
